app/questionnaire/placeholder_parser.py

```diff
@@ -178,6 +180,9 @@ def _parse_transforms(
 
                 transform_args[arg_key] = resolved_value
 
+                if transform["transform"] in TRANSFORMS_REQUIRING_UNRESOLVED_ARGUMENTS:
+                    transform_args["raw_args"] = arg_value
```

Comment:
This would only ever set 1 value. It needs to set the args itself, not just the value. Needs to be moved outside the for loop.